src/env.ts

```diff
@@ -0,0 +1,111 @@
+import dotenv from 'dotenv'
+import * as envalid from 'envalid'
+import { makeValidator } from 'envalid'
+import { singleton } from 'tsyringe'
+
+if (process.env.NODE_ENV === 'test') {
+  dotenv.config({ path: 'tests/test.env' }) //different configs for unit and integration??
```

Comment:
does this file exist? If not it might be worth creating it just so we can set `LOG_LEVEL=silent`